spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -241,7 +242,7 @@
 
     context 'skip address step?' do
       let(:pii_from_user) { flow.flow_session[:pii_from_user] }
-      let(:params) { ActionController::Parameters.new({ state_id: submitted_values }) }
+      let(:params) { ActionController::Parameters.new({ identity_doc: submitted_values }) }
       let(:enrollment) { InPersonEnrollment.new }
       let(:identity_doc_address_state) { 'Nevada' }
       let(:identity_doc_city) { 'Twin Peaks' }
```

Comment:
I think you will need to update spec/support/features/in_person_helper.rb as well